lib/utils/data/dap_index.dart

```diff
@@ -112,7 +112,7 @@ Widget appInfoPage(String title, Color accentColor, PackageManifest manifest) {
                 Container(
                   height: 16,
                 ),
-                RaisedButton.icon(
+                ElevatedButton.icon(
```

Comment:
is this styled like the old elevatedbutton implementation?

Reply:
Idk how the old `ElevatedButton` used to look but right know it is the recommended replacement for `RaisedButton`.

Here is a quick comparison:

![image](https://user-images.githubusercontent.com/51208565/148583872-16f17420-e202-4553-97f1-4684499413bb.png)